packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -11,26 +11,27 @@
  */
 
 .root {
-  @include border-radius($base-border-radius);
+  @include border-radius(var(--BorderRadius));
 
   display: inline-block;
   position: relative;
-  margin-block: 0;
-  margin-inline: 0;
-  padding-block: $spacing-xs-em;
-  padding-inline: $spacing-s-em;
+  margin-block: var(--SpaceMarginBlock);
+  margin-inline: var(--SpaceMarginInline);
+  padding-block: var(--SpacePaddingBlock);
+  padding-inline: var(--SpacePaddingInline);
   transition-property: color, background-color, border-color, box-shadow;
-  transition-duration: 100ms;
-  transition-timing-function: linear;
-  border: 1px solid transparent;
+  transition-duration: var(--TransitionDuration);
+  transition-timing-function: var(--TransitionTiming);
+  border-width: var(--BorderWidth);
+  border-style: var(--BorderStyle);
+  border-color: var(--ColorBorderPrimary);
   background-color: var(--ColorBackgroundPrimary);
-  box-shadow: 0 0 0 0 $color-primary-light;
```

Comment:
need to double check if this style is safe to remove

Reply:
so far seems the answer is yes given my manual QA but will check VRT too. With all 0 values for box-shadow here it won't be visible despite the color being assigned here so I think this is safe to remove

Reply:
I believe this is so that the `box-shadow` grows during the transition, rather than appearing suddenly.

Reply:
@edburyenegren-okta @andrewberg-okta I checked on this and the button focus is working just fine without the zeroed out box-shadow style here: https://fe466a7.ods.dev/?path=/story/components-button--primary